apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/DatasetsResource.java

```diff
@@ -115,7 +121,13 @@ public Response findDatasets(
                 .name(name)
                 .build();
 
-        return Response.ok(service.find(page, size, criteria)).build();
+        String workspaceId = requestContext.get().getWorkspaceId();
+
+        log.info("Finding datasets by '{}' on workspaceId '{}'", criteria, workspaceId);
+        DatasetPage datasetPage = service.find(page, size, criteria);
```

Comment:
For future reference, just be careful about not leaking sensitive data to the logs in objects like this criteria. In this particular case, it should be fine.

Reply:
Agree. I thought about it, but I assumed those were not sensitive